web_ui/prometheus_test.go

```diff
@@ -169,7 +171,7 @@ func TestPrometheusProtectionOriginHeaderScope(t *testing.T) {
 
 	//Setup a private key and a token
 	viper.Set("IssuerKey", kfile)
-
+	viper.Set("ConfigDir", t.TempDir())
```

Comment:
For some reason, occasionally t.TempDir() is cleaned up before a process is done using the directory. This is an issue in windows as windows does not allow the removal of files/directories if a process is using it. To fix this, simply replace the t.TempDir() with an os.MkDirTemp() and add a defer os.RemoveAll(<temp dir path>). The defer happens in a place windows typically likes.